cmd/node/node.go

```diff
@@ -547,6 +549,27 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 	return nil
 }
 
+// periodically checks that our clock is sync
+func (app *SpacemeshApp) checkTimeDrifts() {
+	checkTimeSync := time.NewTicker(app.Config.TIME.RefreshNtpInterval)
+	defer close(app.term)      // signal checkTimeDrifts closed
+	defer checkTimeSync.Stop() // first close ticker
```

Comment:
also here defer is not really needed since eventually this will be called only in the case `<-app.term`. you can just call stop and close there and return instead of break